src/main/java/net/foxdenstudio/sponge/foxguard/plugin/handler/WelcomeHandler.java

```diff
@@ -119,6 +177,16 @@ public void save(Path directory) {
         root.getNode("exit").setValue(exitTemplate);
     }
 
+    private static TextTemplate fromString(String string) {
+        TextTemplate template = TextTemplate.of();
+
+        for(String object : pattern.split(string)) {
```

Comment:
So this.
This is fantastic.
You are now part of the java regex club.

String.split REMOVES anything it matches, which means running split with the regex I gave you will give you an array of empty strings.